experimental/generation/generator/templates/booleanEntity.en-us.lg.lg

```diff
@@ -5,7 +5,7 @@
 
 # template
 - ```
-${'#'} booleanEntity_DisplayValue(value)
+${'#'} getEntityValueText_booleanEntity(value)
```

Comment:
>getEntityValueText_booleanEntity [](start = 7, length = 32)

You have put the property/entity last.  When we had talked before we had put that first so that you can find easily get completion on all the templates related to a property/entity.

Reply:
yeah, switched back.

---
In reply to: [433366141](https://github.com/microsoft/BotBuilder-Samples/pull/2461#discussion_r433366141) [](ancestors = 433366141)